client/components/tinymce/plugins/simple-payments/dialog/index.jsx

```diff
@@ -27,27 +26,34 @@ class SimplePaymentsDialog extends Component {
 		isEdit: PropTypes.bool.isRequired,
 		onChangeTabs: PropTypes.func.isRequired,
 		onClose: PropTypes.func.isRequired,
+		onInsert: PropTypes.func.isRequired,
```

Comment:
When switching to Editor HTML mode, I'm getting React props warnings:

![image](https://user-images.githubusercontent.com/4988512/28169228-f8ebc3fc-67e1-11e7-9045-8e03da601872.png).

Check `client/post-editor/editor-html-toolbar/index` -- `SimplePaymentsDialog` is also used there and it represents the HTML mode of Editor.